templates/ImageBuilder/cis-for-eb-image-pipeline-template.yaml

```diff
@@ -0,0 +1,138 @@
+  # AWS Systems Manager (SSM) and EC2 Image Builder the necessary permissions to build an image.
+  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html
+  # https://docs.aws.amazon.com/imagebuilder/latest/userguide/image-builder-setting-up.html
+  InstanceRole:
+    Type: AWS::IAM::Role
+    Metadata:
+      Comment: Role to be used by instance during image build.
+    Properties:
+      ManagedPolicyArns:
+        - Fn::Sub: arn:${AWS::Partition}:iam::aws:policy/AmazonSSMManagedInstanceCore
+        - Fn::Sub: arn:${AWS::Partition}:iam::aws:policy/EC2InstanceProfileForImageBuilder
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action:
+              - sts:AssumeRole
+            Effect: Allow
+            Principal:
+              Service:
+                - !Sub 'ec2.${AWS::URLSuffix}'
+        Version: '2012-10-17'
+      Path: /executionServiceEC2Role/
+  # To pass the InstanceRole to an EC2 instance, we need an InstanceProfile.
+  # This profile will be used during the image build process.
+  # https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html
+  InstanceProfile:
+    Type: AWS::IAM::InstanceProfile
+    Properties:
+      Path: /executionServiceEC2Role/
+      Roles:
+        - Ref: InstanceRole
+
+  ImageBuilderImagePipeline:
+    Type: "AWS::ImageBuilder::ImagePipeline"
+    Properties:
+      Name: !Sub ${AWS::StackName}
+      Status: "ENABLED"
+      Schedule:
+        ScheduleExpression:
+          rate(7 days)
+      InfrastructureConfigurationArn:
+        Ref: ImageInfrastructureConfiguration
+      ImageScanningConfiguration:
+        ImageScanningEnabled: true
+      ImageRecipeArn:
+        Ref: Recipe
+      DistributionConfigurationArn:
+        Ref: ImageDistributionConfiguration
+      ImageTestsConfiguration:
+        TimeoutMinutes: 720
+        ImageTestsEnabled: true
+
+  Recipe:
+    Type: "AWS::ImageBuilder::ImageRecipe"
+    Properties:
+      Name: !Sub ${AWS::StackName}
+      Version: !Ref ImageVersion
+      ParentImage:
+        Fn::Sub: arn:${AWS::Partition}:imagebuilder:${AWS::Region}:aws:image/cis-hardened-image-level-1-on-amazon-linux-2023-arm-r5sqt5uxgygqi/x.x.x
+      Components:
+        - ComponentArn:
+            Ref: CISforEBImageBuilderComponent
+      # The image will not build with the default working dir, /tmp, since CIS makes that directory non-executable
+      WorkingDirectory: "/root"
+      BlockDeviceMappings:
+      - Ebs:
+          Throughput: 125
+          VolumeType: gp3
+          Iops: 3000
+          VolumeSize: !Ref VolumeSize
+          DeleteOnTermination: true
+        DeviceName: "/dev/xvda"
+      AdditionalInstanceConfiguration:
+        SystemsManagerAgent:
+          UninstallAfterBuild: false
+
+  CISforEBImageBuilderComponent:
+    Type: AWS::ImageBuilder::Component
+    Properties:
+      Name: CISforEBImageBuilderComponent
```

Comment:
it's usually not good to have a static name, to reduce conflict i suggest you set this to `${StackName}` or prepend it to this value.